daemon/src/main/java/org/apache/maven/cli/DaemonMavenCli.java

```diff
@@ -473,6 +472,7 @@ DefaultPlexusContainer container() throws Exception {
 
         List<File> extClassPath = Stream.of(
                         Environment.MVND_EXT_CLASSPATH.asString().split(","))
+                .filter(s -> s != null && !s.isEmpty())
```

Comment:
The missing empty filter here lead to the empty String being added to the extClassPath list, which made the classloader structure slightly different from vanilla Maven in some scenarios. Not a big deal in practice, but it broke some of our tests, so I decided to fix it for consistency :)